src/main/kotlin/com/zepben/evolve/services/network/NetworkService.kt

```diff
@@ -67,21 +74,24 @@ class NetworkService : BaseService("network") {
     fun add(overheadWireInfo: OverheadWireInfo): Boolean = super.add(overheadWireInfo)
     fun remove(overheadWireInfo: OverheadWireInfo): Boolean = super.remove(overheadWireInfo)
 
+    fun add(potentialTransformerInfo: PotentialTransformerInfo): Boolean = super.add(potentialTransformerInfo)
+    fun remove(potentialTransformerInfo: PotentialTransformerInfo): Boolean = super.remove(potentialTransformerInfo)
+
```

Comment:
todo: You have merged these into the wrong package subsection. Should be under `IEC61968 infIEC61968 InfAssetInfo`